src/server/index.ts

```diff
@@ -40,6 +40,10 @@ const build = () => {
     } catch (error) {
       server.log.error(error)
 
+      if (error?.name === 'SyntaxError') {
+        return reply.code(422).send(error)
```

Comment:
I'm not sure if this should be a 422 or just a 400...

Reply:
Changing it